src/Client/AzureManaged/DurableTaskSchedulerClientExtensions.cs

```diff
@@ -40,15 +40,70 @@ public static void UseDurableTaskScheduler(
+    /// <param name="configure">Optional callback to dynamically configure DurableTaskSchedulerClientOptions.</param>
+    public static void UseDurableTaskScheduler(
+        this IDurableTaskClientBuilder builder,
+        string endpointAddress,
+        string taskHubName,
+        TokenCredential credential,
+        DurableTaskSchedulerClientOptions.ClientRetryOptions retryOptions,
+        Action<DurableTaskSchedulerClientOptions>? configure = null)
+    {
+        ConfigureSchedulerOptions(
+            builder,
+            options =>
+            {
+                options.EndpointAddress = endpointAddress;
+                options.TaskHubName = taskHubName;
+                options.Credential = credential;
+                options.RetryOptions = retryOptions;
+            },
+            configure);
+    }
+
+    /// <summary>
+    /// Configures Durable Task client to use the Azure Durable Task Scheduler service using a connection string.
+    /// </summary>
+    /// <param name="builder">The Durable Task client builder to configure.</param>
+    /// <param name="connectionString">The connection string used to connect to the Durable Task Scheduler service.</param>
+    /// <param name="configure">Optional callback to dynamically configure DurableTaskSchedulerClientOptions.</param>
+    public static void UseDurableTaskScheduler(
+        this IDurableTaskClientBuilder builder,
+        string connectionString,
+        Action<DurableTaskSchedulerClientOptions>? configure = null)
+    {
+        var connectionOptions = DurableTaskSchedulerClientOptions.FromConnectionString(connectionString);
+        ConfigureSchedulerOptions(
+            builder,
+            options =>
+            {
+                options.EndpointAddress = connectionOptions.EndpointAddress;
+                options.TaskHubName = connectionOptions.TaskHubName;
+                options.Credential = connectionOptions.Credential;
+                options.AllowInsecureCredentials = connectionOptions.AllowInsecureCredentials;
+            },
+            configure);
+    }
+
     /// <summary>
     /// Configures Durable Task client to use the Azure Durable Task Scheduler service using a connection string.
     /// </summary>
     /// <param name="builder">The Durable Task client builder to configure.</param>
     /// <param name="connectionString">The connection string used to connect to the Durable Task Scheduler service.</param>
+    /// /// <param name="retryOptions">The options that determine how and when a request will be retried.</param>
     /// <param name="configure">Optional callback to dynamically configure DurableTaskSchedulerClientOptions.</param>
     public static void UseDurableTaskScheduler(
         this IDurableTaskClientBuilder builder,
         string connectionString,
+        DurableTaskSchedulerClientOptions.ClientRetryOptions retryOptions,
```

Comment:
Same comment as above. I don't think we need a new `UseDurableTaskScheduler` method overload for this. The existing methods should be usable as-is.

Reply:
See above.